src/lib/molecules/ExtendedInput/index.js

```diff
@@ -154,7 +154,7 @@ const ExtendedInput = forwardRef((props, ref) => {
             if (isDropdown) {
                 onClick(e);
                 if (!readOnly) {
-                    stopEvent(e, true);
+                    e.preventDefault();
```

Comment:
As I know this method is used as an `onMouseDown` event handler for the `Icon` component. 
The `event.preventDefault()` prevents the default action associated with the event from occurring.
In this case, the `onMouseDown` event applied on the `<i>` tag inside of the `Icon` component which has no default action to prevent it. It means you can remove event prevention logic. Please check and come back with an answer.